homeassistant/components/seventeentrack/strings.json

```diff
@@ -0,0 +1,40 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+    },
```

Comment:
We're missing abort reasons for cannot_connect and invalid_credentials.

Reply:
added